collect_app/build.gradle

```diff
@@ -191,12 +192,18 @@ dependencies {
     implementation "com.android.support:exifinterface:${rootProject.supportLibraryVersion}"
     implementation "com.android.support:multidex:1.0.3"
 
-    implementation "com.google.android.gms:play-services-analytics:${rootProject.playServicesVersion}"
-    implementation "com.google.android.gms:play-services-auth:${rootProject.playServicesVersion}"
-    implementation "com.google.android.gms:play-services-maps:${rootProject.playServicesVersion}"
-    implementation "com.google.android.gms:play-services-location:${rootProject.playServicesVersion}"
-
-    implementation "com.google.firebase:firebase-core:${rootProject.playServicesVersion}"
+    /*As beginning with version 15 of all Play services and Firebase libraries,each Maven dependency
+     matching com.google.android.gms:play-services-* and com.google.firebase:firebase-* is no longer
```

Comment:
I think once this comment is within the PR description we are good and it doesn't have to be here as the maintainers will know of this and when changes are happening versioning wise it will always have to be validated anyways.

Reply:
ok I removed that comment.